PEPit/functions/convex_indicator.py

```diff
@@ -9,7 +9,7 @@ class ConvexIndicatorFunction(Function):
     implementing interpolation constraints for the class of closed convex indicator functions.
 
     Attributes:
-        D (float): upper bound on the diameter of the feasible set
+        D (float): upper bound on the diameter of the feasible set, that can be infinite
```

Comment:
that can be infinite -> ce n'est pas formel.

remplacer par quelque chose du genre "(possibly set to np.inf)" ?